tests/lib/rules/jsx-uses-react.js

```diff
@@ -35,50 +35,50 @@ const settings = {
 // -----------------------------------------------------------------------------
 
 const ruleTester = new RuleTester({ parserOptions });
-const linter = ruleTester.linter || eslint.linter || eslint.Linter;
-linter.defineRule('jsx-uses-react', require('../../../lib/rules/jsx-uses-react'));
+const ruleDefiner = ruleTester[Symbol.for('react.RuleTester.RuleDefiner')] || ruleTester.linter || eslint.linter || eslint.Linter;
+ruleDefiner.defineRule('react/jsx-uses-react', require('../../../lib/rules/jsx-uses-react'));
 
 ruleTester.run('no-unused-vars', rule, {
   valid: parsers.all([
-    { code: '/*eslint jsx-uses-react:1*/ var React; <div />;' },
-    { code: '/*eslint jsx-uses-react:1*/ var React; (function () { <div /> })();' },
-    { code: '/*eslint jsx-uses-react:1*/ /** @jsx Foo */ var Foo; <div />;' },
+    { code: '/*eslint react/jsx-uses-react:1*/ var React; <div />;' },
+    { code: '/*eslint react/jsx-uses-react:1*/ var React; (function () { <div /> })();' },
+    { code: '/*eslint react/jsx-uses-react:1*/ /** @jsx Foo */ var Foo; <div />;' },
```

Comment:
Adding a plugin prefix is needed for ESLint v9 RuleTester, but this works with older versions as well, and I think the tests look more realistic this way anyway.